src/models/FilterString.js

```diff
@@ -11,8 +11,10 @@ import * as contexts from './Contexts';
 class StringItem {
   constructor({
     uid = '',
+    checked = true,
   }) {
     this.uid = uid;
+    this.checked = Boolean(checked);
```

Comment:
Is it used?

Reply:
Yes, in FilterMonitor finally, to check/unckeck items temporarily:

<img width="308" alt="Screenshot 2020-03-16 at 09 25 59" src="https://user-images.githubusercontent.com/1181642/76736984-2f892c00-6768-11ea-8dc7-de7b91078731.png">

Reply:
Right. It would be great to move this temporary state into the component at some point.

Reply:
yep I saw you started doing that, listing all the `excludedIds`